### PR TITLE
Update outdated dependencies: babel/core, mini-css-extract-plugin, eslint, webpack, webpack-cli, less-loader, lhci/cli

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Update outdated dependencies: babel/core, mini-css-extract-plugin, eslint, webpack, webpack-cli, less-loader, lhci/cli
- Re-generated CSS/JS from design system updates.

## Testing

1. Running yarn && yarn serve should work.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
             <title>consumerfinance.gov lighthouse reports</title>
         
         
-            <link rel="stylesheet" href="/cfgov-lighthouse/static/css/main.css">
+            <link rel="stylesheet" href="/static/css/main.css">
```
Comment (wpears) [inline docs/index.html:11]: These path changes seem unrelated to the rest of the PR, is this intended?

Comment (anselmbradford) [reply docs/index.html:11]: Yeah it's a difference in our local setup versus not and were generated with `yarn && yarn start`, but @chosak tells me committing them doesn't matter.

**Review by wpears — APPROVED** [review]
